packages/react/src/components/Forms/Select/mono.spec.tsx

```diff
@@ -317,7 +456,10 @@ describe("<Select/>", () => {
                   },
                 ]}
                 value={value}
-                onChange={(e) => setValue(e.target.value as string)}
+                onChange={(e) => {
+                  setValue(e.target.value as string);
```

Comment:
A bit out of the scope, but I can see `value` can be from type `string | number | string[] | undefined` when `Option.value` can only be with `string | undefined`(https://github.com/openfun/cunningham/blob/main/packages/react/src/components/Forms/Select/mono.tsx#L9), I suppose they should have the same type and if possible infer from each other to avoid casting, what do you think ?

Reply:
Yes I agree with you, we took the constraint to have the exact type as the native `<select/>` which uses this union. However I agree that there is maybe something to work on on this. 

+ I think having `value` that could only be of type `string` is a mistake has it could be useful to allow `number` too